checkstyle.xml

```diff
@@ -61,7 +61,7 @@
         <module name="SimplifyBooleanExpression" />
         <module name="SimplifyBooleanReturn" />
         <module name="StringLiteralEquality" />
-        <module name="AvoidStarImport" />
+        <!--<module name="AvoidStarImport" />-->
```

Comment:
Let's discuss that based on the arguments from https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad